### PR TITLE
Clarified some stuff in the intro section

### PR DESCRIPTION
**Pull request by vikeri**
Hopefully made it somewhat more beginner friendly.

* Added disclaimer that terminal commands are for unix
* Updated the terminal commands to create, edit and run an elixir file

### CODE REVIEW

**Comment by eksperimental** [comment]
Whats the point of using touch? If we are using > later

**Comment by josevalim** [comment]
Thank you! What if we change to not use terminal commands then? Just say: "imagine you have a file named this with these contents.... you can run it with ..." and that's it? Great catch.

**Comment by vikeri** [comment]
I added `touch` to clarify the process: 
1. Create file
2. Edit the file
3. Run file.

But it could of course be removed.

I still think that it is good to make people use the terminal both because this part of the tutorial should be agnostic to tooling like editor choice etc. and also because if you don't already know these terminal commands it is good to get exposed to them.

**Comment by josevalim** [comment]
> if you don't already know these terminal commands it is good to get exposed to them.

There are many other important skills that I'd say a developer needs to get exposed to. We could for example make people acquainted big-O notation, that's not the point of the this tutorial though. :) Specially when the commands are OS specific.

If you could amend your PR to not rely on the terminal commands, we would really appreciate it. Otherwise one of us can get to it later. :) Thank you!

**Comment by vikeri** [comment]
Sure I'll update it :) but just to clarify (no WIndows computer nearby); how do I run elixir on windows? `elixir.exe simple.exs` or just `elixir simple.exs` or some other variant? And should I add hints that if elixir can't be found that it must be added to the path: http://elixir-lang.org/install.html#setting-path-environment-variable

**Comment by josevalim** [comment]
`elixir simple.exs` should be enough. Don't worry about elixir not being on the path given it is covered in the install guide. :)

**Comment by vikeri** [comment]
@josevalim Updated PR now

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: getting-started/introduction.markdown [hunk 1]
```diff
@@ -48,18 +48,21 @@ It seems we are ready to go! We will use the interactive shell quite a lot in th
...
-```bash
-$ cat simple.exs
-IO.puts "Hello world
-from Elixir"
+```elixir
+IO.puts "Hello world from Elixir"
+
+```
 
+ Save it as `simple.exs` and execute it with `elixir`:
```
Comment (eksperimental) [inline getting-started/introduction.markdown]: Wouldnt it be concenient as well to link here to a simple.exs file so people can download directly

**Pull request merged**